package.json

```diff
@@ -30,8 +29,7 @@
     "deploy:package": "scripts/publish.sh",
     "contributors:update": "node scripts/update-contributors.js",
     "dev": "styleguidist server --config config/styleguide.config.js",
-    "dev:e2e": "NODE_ENV=e2e docker-compose up ciboot",
-    "dev:e2e-direct": "NODE_ENV=e2e webpack-dev-server --config e2e/visual/webpack.config.js",
```

Comment:
Can we keep `dev:e2e-direct` for local testing?

Reply:
`dev:e2e-direct` is now `dev:e2e`. Testing via a local server causes issues with saucelabs at times, so we're avoiding keeping that functionality and removing almost all docker usage (as it says in the ticket)